source/manual/get-started.html.md

```diff
@@ -90,9 +90,8 @@ If you have a YubiKey, you will use `gpg-agent` in place of `ssh-agent`, which r
 
 ## 3. Set up your GitHub account
 
-1. Set up a [GitHub] account (or use your existing personal account).
+1. [Login into your existing GitHub account][github-login] or [create a new GitHub account][github-signup].
 1. [Associate your GitHub account with your GDS email address][associate-email-github], which can be in addition to your personal email address.
```

Comment:
Do we need a step to tell people to enable 2-factor for their GitHub accounts? Or are we already enforcing that at org level so it'll make them set it up anyway?

Reply:
Yup enforced at a org level